theme/homepage.tsx

```diff
@@ -0,0 +1,15 @@
+import type { Culture } from '@prezly/sdk';
+
+import { api } from '@/theme-kit';
+
+interface Match {
+    locale: Culture['code'];
+}
+
+export default async function Page({ locale }: Match) {
+    const { contentDelivery } = api();
+
+    const language = (await contentDelivery.language(locale))!;
```

Comment:
Not a big fan of this non-null assertion.
Can we make a separate endpoint for validating the locale (where it's not yet clear if that's an enabled locale), and another one for scenarios we're confident that locale is enabled (like here)?

Reply:
I agree. I just didn't spend time on typescript gymnastics, leaving it for later. I'll definitely look for ways to avoid these.

Reply:
dc6e0c79c44eae36933d9b3ad9f6657d1e985cca